docs/RELEASE_NOTES.md

```diff
@@ -146,287 +147,253 @@ Use Cocos Console to create a new game:
 
-## The main features in detail of Cocos2d-x v3.9:
+## The main features in detail of Cocos2d-x v3.10:
 
-### 3D Module
+### UI System
 
-1. 3D MotionStreak
+1. Reimplemented Scale9Sprite and improve the scale9sprite performance and reduce memory consumption.
     
-    In this version, 3D MotionStreak is added to support streak effect. Check the testcase: [Sprite3DTest](https://github.com/cocos2d/cocos2d-x/blob/v3/tests/cpp-tests/Classes/Sprite3DTest/Sprite3DTest.cpp#L2472) to see how to use it.
+    Reimplemented ui::Scale9Sprite, now the Slice sprite uses 16 vertices and 54 indices instead of the old 9 sprite way, The memory consumption is much lower than the previous implementation, and it is also more time efficient.
 
-2. Sprite3D
+    In SIMPLE mode, the 4 borders are all 0 and the whole sprite will scale horizontally and vertically. In this mode only 1 quad is used to rendering, for example:
 
-    Added Sprite3D material class. It will be easy and convenient to create internal material. 
+        auto blocks = ui::Scale9Sprite::createWithSpriteFrameName("blocks9c.png");
+        //When setting to SIMPLE, only 4 vertexes is used to rendering.
+        blocks->setRenderingType(Scale9Sprite::RenderingType::SIMPLE);
 
-### 2D Module
+    In SLICE mode, it will use 18 triangles to rendering the slice 9 sprite. If the 4 borders are 0, there still be 18 triangles computed. So choose your RenderingType wisely,for example:
 
-1. Frame callback function and animation callback function
-
-    Three interfaces are added in ActionTimelineData class, which are addFrameEndCallFunc, removeFrameEndCall and clearFrameEndCalls. It will be easy to add or remove specific frame event. 
-
-2. Script Component 
-
-    Script component is used to extend c++ Nodes. You can add a script component to a Node, then the script component will receive onEnter,onExit and update events. For example:
-
-    ```c++ 
-    // create a sprite and add a lua component auto player =
-    Sprite::create("player.png");
-    
-    auto luaComponent = ComponentLua::create("player.lua");
-    player->addComponent(luaComponent);
-    ```
-    
-    ```lua
-    // player.lua
-    local player = { 
-        onEnter = function(self)
-        -- do some things in onEnter 
-        end
-        
-        onExit = function(slef) 
-        -- do some things in onExit 
-        end
-        
-        update = function(self)
-        -- do some things every frame 
-        end
-    }
-    
-    -- it is needed to return player to let c++ nodes know it 
-    return player 
-    ```
+		auto sprite = ui::Scale9Sprite::createWithSpriteFrameName("blocks9c.png");
+        //When setting to SLICE, 16 vertexes will be used to rendering.
+        sprite->setRenderingType(Scale9Sprite::RenderingType::SLICE);
     
-    Javascript can work as the same way, just use ComponentJS instead of ComponentLua. 
+2. Changed PageView to derived from ListView, PageView can be added any widget as child.
```

Comment:
Changed PageView to derived from ListView, PageView can add any type of widget as child.